src/main/java/com/limechain/network/ProtocolUtils.java

```diff
@@ -31,6 +31,10 @@ public static String getKadProtocol(String chainId) {
         return String.format("/%s/kad", chainId);
     }
 
+    public static String getGrandpaProtocol(String chainId, boolean legacyProtocol) {
+        return String.format("/%s/grandpa/1", legacyProtocol ? "paritytech" : chainId);
+    }
+
     public static String getGrandpaProtocol(String chainId) {
```

Comment:
This one seems unused. Can we remove it?

Reply:
It will be used for grandpa substream